app/views/compute_resources_vms/form/_openstack.html.erb

```diff
@@ -12,7 +12,7 @@
 </div>
 <%= select_f f, :tenant_id, compute_resource.tenants, :id, :name, {}, :label => _('Tenant') %>
 
-<%= select_f f, :security_groups, compute_resource.security_groups, :name, :name, {}, :label => _("Security group")  %>
+<%= select_f f, :security_group, compute_resource.security_groups, :name, :name, {}, :label => _("Security group")  %>
```

Comment:
I'm not sure it is, reading fog code imply security_groups https://github.com/fog/fog/blob/master/lib/fog/openstack/models/compute/server.rb#L59